cvprac/cvp_client.py

```diff
@@ -193,23 +194,23 @@ def set_version(self, version):
         '''
         self.version = version
         split_version = version.split('.')
-        print split_version
+        print(split_version)
         # Expect version string to be at least two long
         # Ex: 2018.2
         # Ex: 2018.1.4
         # Ex: 2017.2
         if len(split_version) > 2:
             # Set apiversion to v2 for 2018.2 and beyond.
             if int(split_version[0]) > 2017 and int(split_version[1]) > 1:
-                print 'FOUND V2'
+                print('FOUND V2')
                 self.apiversion = 'v2'
             else:
-                print 'FOUND V1'
+                print('FOUND V1')
                 self.apiversion = 'v1'
         else:
             # If version is shorter than 2 elements for some reason default
             # to v2
-            print 'SHORT VERSION STRING SET v2'
+            print('SHORT VERSION STRING SET v2')
```

Comment:
Could you revert changes to cvp_client.py? I've converted these print statements to log.info statements in the latest release and develop branch.

Reply:
I certainly can, but the cvprac package's installation fails on my macOS laptop with python3 without the parentheses

Reply:
The print statements are removed in the latest release 1.0.0. Can you try installing that one and let me know if that fails? The only remaining prints are the ones in the docs.